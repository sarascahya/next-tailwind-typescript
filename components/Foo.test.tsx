import React from 'react';
import { shallow } from 'enzyme';

import Foo from './Foo';

describe('<Foo />', () => {
  it('renders <Foo /> component', () => {
    const wrapper = shallow(<Foo />);
    expect(wrapper).toMatchSnapshot();
  });
});
