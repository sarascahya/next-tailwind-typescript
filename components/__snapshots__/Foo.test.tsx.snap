// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Foo /> renders <Foo /> component 1`] = `ShallowWrapper {}`;
